audio/Makefile

```diff
@@ -0,0 +1,156 @@
+#
+# Copyright (C) 2009-2012 Chris McClelland
+# Copyright 2015 Joel Stanley <joel@jms.id.au>
+# Copyright 2017 Kyle Robbertze <krobbertze@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# To build the firmware you will need:
+#   SDCC from http://sdcc.sourceforge.net
+#   Git from https://git-scm.com/
+#
+# To build a firmware suitable for loading into RAM:
+#   make
+#
+# To build a firmware suitable for loading from EEPROM:
+#   make FLAGS="-DEEPROM"
+#   
+# To load a firmware:
+# 	make load
+# 	You will need HDMI2USB-mode-switch from
+# 	https://github.com/timvideos/HDMI2USB-mode-switch
+#
+# TODO: Use a common Makefile between all the firmware versions
+TARGET := audio
+FX2LIBDIR := ../third_party/fx2lib
+LINUXHEADERSDIR := ../third_party/fx2lib-linux-headers
+LIBS := $(FX2LIBDIR)/lib/fx2.lib
+INCS := -I sdcc -I$(FX2LIBDIR)/include -I$(LINUXHEADERSDIR) -I.
+
+# Settings specific for the TimVideo hdmi2usb firmware
+BOARD ?= fx2
+FLAGS +=-DDEBUG -DBOARD_$(BOARD)
+USE_16K := 1
+
+# Must only be hex numbers
+FIRMWARE_VERSION := $(shell date +%Y%m%d)
+
+CC_SRCS := descriptors.c firmware.c
+CC_OBJS := $(CC_SRCS:%.c=%.rel)
+
+CC := sdcc
+
+ifeq (${BOARD},atlys)
+VID := 1D50
+PID := 60B7
+DID := 0002
+else
+ifeq (${BOARD},fx2)
```

Comment:
This shouldn't be using the Opsis values and the DID should be unique for each board.

I've changed it to use the same as the Atlys values which is the OpenMoko IDs assigned to our project.